File: py/IEC61968/Operations/FieldSafetySupervisor.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
from CIM_STD_PYTHON.TC57CIM.IEC61968.Common.CrewMember import CrewMember


class FieldSafetySupervisor(CrewMember):
    def __init__(self):
        super().__init__()
        self.released_safety_documents = None  # All safety documents released by this supervisor
        self.issued_safety_documents = None  # All safety documents issued to this supervisor

File: java/IEC61968/Operations/FieldSafetySupervisor.java
package IEC61968.Operations;

import IEC61968.Common.CrewMember;

/**
 * Crew member on work site responsible for all local safety measures for the work
 * crew doing maintenance, construction and repair in a substation or on a power
 * line/cable.
 * @author T. Kostic
 * @version 1.0
 * @created 25-Dec-2023 8:45:22 PM
 */
public class FieldSafetySupervisor extends CrewMember {

	/**
	 * All safety documents released by this supervisor.
	 */
	public SafetyDocument ReleasedSafetyDocuments;
	/**
	 * All safety documents issued to this supervisor.
	 */
	public SafetyDocument IssuedSafetyDocuments;

	public FieldSafetySupervisor(){

	}
}//end FieldSafetySupervisor